courses/free-courses-en.md

```diff
@@ -1141,6 +1142,11 @@
 * [R Tutorial For Beginners \| Edureka](https://www.youtube.com/watch?v=fDRa82lxzaU) - Edureka!
 
 
+### React-Testing-Library
+
+* [React Testing Library Crash Course](https://www.youtube.com/playlist?list=PL4cUxeGkcC9gm4_-5UsNmLqMosM-dzuvQ) - The Net Ninja
```

Comment:
It's usually not helpful to add a category for a single resource. Try to find another category, or see if there are other books in our lists that would fit into your category.

Reply:
Hello @eshellman,

The category that came close was JavaScript, does it make sense to add it there? Also, you might have missed it, this category is actually for free online courses not books.